packages/lesswrong/components/tagging/FooterTag.tsx

```diff
@@ -14,11 +14,9 @@ import type { TagsTooltipPreviewWrapper } from './TagsTooltip';
 const useExperimentalTagStyleSetting = new DatabasePublicSetting<boolean>('useExperimentalTagStyle', false)
 
 export const tagStyle = (theme: ThemeType): JssStyles => ({
-  marginRight: 3,
   padding: 5,
   paddingLeft: 6,
   paddingRight: 6,
-  marginBottom: 8,
```

Comment:
Not sure why these were removed and if this was supposed to be the case for everything using `tagStyle` (both within a single forum and across forums).  @Discordius ?